### PR TITLE
Chore: Add Readme / Styling touch-ups

### PR DESCRIPTION
**Pull request by ndom91**
Hey Everyone,

Just added a readme with some install instructions and modified the index page a bit to increase whitespace in the hero section and unify the style of the following sections.

### CODE REVIEW

**Comment by vladdoster** [comment]
**Nits**

1. Commits could be squashed into 1.
2. It's easier to review changes by separating formatting changes in a separate PR. Formatting adds a lot of noise.

**Comment by ndom91** [comment]
Yeah good call, the commit history is a bit unruly - could definitely use a squash on merge.

**Comment by vladdoster** [comment]
You can rebase the commit on your own

1. 
    ```bash
    git rebase -i HEAD~6
    ```
2. Change the first commit in the list from `pick` ➡️ `reword`
3. Change all the other commits from `pick` ➡️ `fixup`
4. Could use this as your commit message
    ```markdown
    (maint)

    Hey Everyone,
    
    - Created repository `README.md` with local installation instructions
    - More cohesive styling in `main.css`
    - Increased whitespace in `index.html`
    ```
5. `git push --force origin ndom91-clean-up`

File: static/css/main.css [hunk 1]
```diff
@@ -737,21 +742,80 @@ div.chat-button > * {
...
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 40px 0px;
+    width: 90%;
+    max-width: 62em;
 }
 
-.contribute-card > * {
-    color: #FFF;
+div.contribute-layout i {
```
Comment (vladdoster) [inline static/css/main.css:757]: This adds a lot of complexity to what is relatively simple CSS. How do these changes look on mobile?

Could you explain:
1. why you mix `em`, `rem`, and `px`
2. why 40px? Seems very specific

**Comment by vladdoster** [comment]
Mind posting a screenshot of the website changes? I'm curious what the classes (e.g. `contributor-layout`) look like.

**Comment by marcan** [comment]
The README and the whitespace fixes are fine, but please don't mix together unrelated changes like this in the same PR; it makes reviewing it more complicated. Changes should be squashed into commits that each do one logical thing and stand on their own, and completely unrelated things should be separate PRs.

I'm really confused by the CSS changes here. You say you're unifying the styles, but this adds more code; unified styles should take less code. The color changes to the headers reduce the contrast, the main font ends up an ugly low contrast dark on dark instead of light on dark, and the new design breaks on mobile at narrower widths (while the Community section that you're emulating does not). 

The design intent here was not for the three sections to be identical in layout :)

**Comment by ndom91** [comment]
Okay so here's my thinking, 

1. I wanted to make those bottom sections similar to the top one (contribute and documentation similar to community). i.e.

My suggestion / current status of this PR:
image

2. Regarding the amount of code, your right @marcan, the layout of those bottom sections could definitely have been pulled out and abstracted into one class, for example, instead of three separate classes / sections of code for each section in the markup. 

3. In regard to coloring, I also wanted to stick with the same idea that you had going in the top section (Community) and looked nice there, i.e. the heading in a bit lighter color compared to the base background color of the section. It seems like you were already going in that direction with the bottom two sections (contribute and documentation), however, the background color of the heading of these sections was so light so as to barely be distinguishable from white, respectively, instead of grey and light blue. 

I can appreciate that the contrast between the text and the heading background suffered a bit due to these changes, but they still pass all contrast accessibility guidelines.

image

4. Regarding breaking at lower viewport widths, what do you mean exactly? I made sure to keep the `flex-wrap` behavior to make the button wrap below the text area.

image

File: static/css/main.css (hunk 1 above)
Comment (ndom91) [reply static/css/main.css:757]: I rebased and used your suggested commit message as well as posted some screenshots in my comment below. 

In regard to these "magic numbers", I basically just reused what was already there. The `community-card` class also used a padding of 40px and the wrapper was defined with a max-width of 62em and width of 90%.

**Comment by ndom91** [comment]
Okay so I took all your guys' suggestions to heart and significantly cleaned up that css like marcan hinted that it could be. I was also able to remove a bit of unnecessary css through that. In doing so I was also able to fix the viewport issue. Finally, I tweaked the heading background colors of the sections a bit more towards the original :wink: 

Currently it looks like this:

image

image

Also a bit off topic - but I'd love to put this on vercel/netlify so that we get PR preview deploys and all that nice stuff for free.

**Comment by marcan** [comment]
I think we're running into a design intent misunderstanding here. The other sections are different from the community section, because they're not the same thing. The community section is a list of community mediums, while the other two sections are single items. This is why their design is different. This was a deliberate design decision, not an oversight :)

**Review by vladdoster — COMMENTED** [review]
> Sorry for mixing together the readme and CSS stuff into one PR, I'll keep it more organized in the future. Anyway, here's my thinking:

This mixing of changes shouldn't be put off for "future commits", but fixed for this PR(s). The reason for splitting work into separate PRs is so that in the case something needs to be reverted, it is a simple thing to accomplish. 

For example, if this PR was merged as is and in a week people complain about the CSS and the decision is made to revert CSS changes, marcan wouldn't be able to do that without also reverting the README.md changes or he could but would be a hassle. Ideally, he could revert a commit and that is it. 

As a contributor to an open-source repository, you should always try to reduce the burden on the maintainers and be mindful that you are asking someone to use their brain cycles to review your PR. Reviews will be completed faster if PRs are atomic. In this case that would be separating changes into their own PRs.

**Comment by ndom91** [comment]
@vladdoster yeah so I get why its not good practice, again - sorry. 

Also I see you guys have your own styling and strong opinions about it.. I'll just close this PR and open a new one with only the README addition. Thanks anyway!

Oh also I know you dont haveee to center the logo, but it looks nice. 

This style is taken from [Node.js README.md](https://github.com/nodejs/node)

**Pull request closed without merging**